src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/jira/StandaloneStoppedNode.kt

```diff
@@ -99,6 +103,32 @@ internal class StandaloneStoppedNode(
         return ssh.execute("cat $unpackedProduct/work/catalina.pid").output.trim().toInt()
     }
 
+    private fun waitForRunning(
+        ssh: SshConnection,
+        threadDump: ThreadDump
+    ) {
+        if (!waitForRunning) {
+            logger.debug("Skipping Jira startup wait")
+            return
+        }
+        val statusEndpoint = URI("http://admin:$adminPasswordPlainText@localhost:8080/status")
+        pollJira(
+            launchTimeouts.initTimeout,
+            ssh,
+            threadDump,
+            "RUNNING state"
+        ) {
+            val response = ssh.safeExecute("curl $statusEndpoint", launchTimeouts.unresponsivenessTimeout).output
+            Json.createParser(StringReader(response)).use { jsonParser ->
```

Comment:
I had a choice between `jq` in Bash or `Json` in Java. Both had similar amount of _boilerplate_.